tools/modeling/docs/design_notes/codegen.md

```diff
@@ -0,0 +1,52 @@
+<!--
+  This file was coverted mannually from codegen.html TiddlyWiki.
+  That had serveral pages in it's table of contents, of which only two were not
```

Comment:
```suggestion
  That had several pages in its table of contents, of which only two were not
```

Reply:
Guess I forgot to spell check this file...